tools-local/ship-public-apis/Program.cs

```diff
@@ -109,8 +117,8 @@ static async Task<int> MainAsync(DirectoryInfo path, bool resort)
 
                 lines.Sort(StringComparer.Ordinal);
 
-                await File.WriteAllLinesAsync(shippedTxtPath, lines);
-                await File.WriteAllBytesAsync(unshippedTxtPath.FullName, Array.Empty<byte>());
```

Comment:
Where does this log this data? Is it part of CI/CD or somewhere during build? I haven't seen this yet.

Reply:
No, it's not run as part of the CI build. It's something we need to manually run when we ship a GA version: https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#shipping-nuget